Detectors/ITSMFT/ITS/simulation/src/Detector.cxx

```diff
@@ -183,19 +153,18 @@ static void configITS(Detector *its) {
     nModPerStaveLr = TMath::Nint(tdr5dat[idLr][kNModPerStave]);
     int nChipsPerStaveLr = nModPerStaveLr;
     if (idLr >= kNLrInner) {
-      double modlen = nChipsPerModule*seg0->Dz() + (nChipsPerModule-1)*zChipGap;
+      double modlen = nChipsPerModule*Segmentation::SensorSizeCols + (nChipsPerModule-1)*zChipGap;
       double zlen = nModPerStaveLr*modlen + (nModPerStaveLr-1)*zModuleGap;
       its->defineLayer(idLr, phi0, rLr, zlen, nStaveLr, nModPerStaveLr,
-                       kSiThickOB, seg0->Dy(), seg0->getChipTypeID(), kBuildLevel);
+                       kSiThickOB, Segmentation::SensorThickness, kSensTypeID, kBuildLevel);
```

Comment:
Hi @mario6829 , I've substituted the "ideal" epitaxial layer thickness of 18\mum by the real thickness (45\mum) where the charge needs to be collected from, could you please check if this is correctly handled by the geometry (no protrusions, overlaps etc,)?